jsapp/js/components/list.es6

```diff
@@ -275,8 +275,10 @@ class ListSearchSummary extends React.Component {
     this.setState(state);
   }
   render () {
-    var messages = [], modifier,
-        s = this.state;
+    var messages = [];
```

Comment:
It's not a big deal for this PR, but in general, let's avoid making these types of stylistic changes.